src/revlanguage/main.cpp

```diff
@@ -28,14 +28,38 @@ using po::variables_map;
 
 std::string usage()
 {
-    return "Usage: rb [OPTIONS]\n       rb [OPTIONS] <file1> [<file2> ...]\n";
-    // Other usages not mentioned
+    std::string usage =
+        "Usage: rb [OPTIONS]\n"
+        "       rb [OPTIONS] <file1> [<file2> ...] [--args <arg1> [<arg2> ...]]";
+
+    return usage;
+}
+
+
+std::string usage_examples()
+{
+    std::string usage_examples =
+        "Usage examples:\n"
+        "   1. rb\n"
+        "   2. rb --args 1 2\n"
+        "   3. rb script.Rev\n"
+        "   4. rb --file script.Rev                         # equivalent to 3\n"
```

Comment:
For example, `4. rb --file script.Rev` seems unnecessary, since the recommended form is #3.  It does explain how the parser works, but it actually seems like a counter-example or anti-pattern.

Reply:
Removed